include/util/typedefs.hpp

```diff
@@ -50,8 +50,10 @@ struct duplicated_node
 };
 } // namespace tag
 using OSMNodeID = osrm::Alias<std::uint64_t, tag::osm_node_id>;
+// NOLINTNEXTLINE(bugprone-throw-keyword-missing)
```

Comment:
Are these needed? I can omit these with clang 11.

Doesn't appear to match the [description](https://releases.llvm.org/11.0.0/tools/clang/tools/extra/docs/clang-tidy/checks/bugprone-throw-keyword-missing.html) either.